src/core/css/SearchResult.scss

```diff
@@ -23,7 +23,15 @@
 }
 
 .SearchResult-main {
+  display: flex;
+  flex-direction: row;
   flex-grow: 1;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.SearchResult-text {
+  width: 90%;
```

Comment:
Where did you arrive at these percentages? We tend to use `px` values in the app and not `%`.